pkg/storaged/clevis-luks-passphrase.sh

```diff
@@ -21,7 +27,11 @@ if cryptsetup isLuks --type luks1 "$DEV"; then
     luksmeta show -d "$DEV" | while read slot state uuid; do
         if [ "$state" == "active" -a "$uuid" == "$CLEVIS_UUID" ]; then
             if pp=$(luksmeta load -d "$DEV" -s "$slot" | clevis decrypt); then
-                echo $pp
+                if [ "$type" = yes ]; then
+                    echo clevis
```

Comment:
In this case the just computed `$pp` is entirely ignored. Does this actually have to call luksmeta|clevis in that  case, i.e. does that have a side effect?

Reply:
Yeah, I was asking myself the same... the main point of the `--type` option is to avoid sending the actual passphrase back to the browser and smear all over the JavaScript heap.

The guarantee we need is that if this script says "clevis", we can assume that `clevis luks unlock` will succeed. This doesn't only need a configured key slot, but we also need to be in an environment where it will work (tang server reachable, etc).

Reply:
I think we need to call `clevis decrypt`, `clevis luks unlock -t SLT` does the same. So I wouldn't change anything here.